otter/integration/lib/test_cloud_load_balancer.py

```diff
@@ -235,6 +238,123 @@ def add(clb, clock):
         self.assert_mutate_function_does_not_retry_if_not_pending_update(
             add, main_treq_args)
 
+    def get_fake_treq_for_delete(self, get_json_body, get_response=None,
+                                 del_response=None):
+        """
+        Return a CLB for use with deleting a CLB - this is different than
+        the one returned by `get_clb` because it requires stubbing out two
+        treq requests.
+        """
+        get_response = get_response or Response(200)
+        del_response = del_response or Response(202)
+
+        class FakeTreq(object):
+            def delete(cls, _url, *args, **kwargs):
+                # args and kwargs are the same as the get ones
+                self.assertEqual(args, ())
+                self.assertEqual(kwargs, self.expected_kwargs)
+                self.assertEqual(_url, 'clburl/loadbalancers/{0}'.format(
+                    self.clb_id))
+                return succeed(del_response)
+
+            def get(cls, _url, *args, **kwargs):
+                cls.delete(_url, *args, **kwargs)
+                return succeed(get_response)
+
+            def content(cls, resp):
+                # If the get_response is not a 200, then, it would be
+                # considered an error and hence treq.content would be called.
+                # Otherwise, treq.content is only called for the del response.
+                if get_response == Response(200):
```

Comment:
I am really sorry but this `if` still confuses me. Is it possible to add `content` to `Response` defaulting to empty string? 
```python
class Response(object):
    """Fake response object"""
    def __init__(self, code, headers={}, content=''):
        self.code = code
        self.headers = headers
        self.content = content
```
Then `treq.content` and `.json_content` can have default implementation that returns the content from here. I can understand if you don't want to do this since you are already implementing proper Stub in treq itself.